coralproject/talk
# Bump package json version to 7.2.0

## What does this PR do?

Bump package json version to 7.2.0

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indices.

## How do I test this PR?

- Build Coral
- Check admin
- See new version number on login screen / bottom of admin pages 
 
## How do we deploy this PR?

No special considerations.
